vocab/security/vocabulary.yml

```
@@ -39,9 +39,18 @@ class:
       - label: vc-data-integrity
         url: https://www.w3.org/TR/vc-data-integrity/#dataintegrityproof
 
+  - id: Multikey
+    label: Multikey Verification Method
+    upper_value: sec:VerificationMethod
+    comment: Verification method to be used with the eddsa-2022 cryptographic suite. See the [EdDSA Cryptosuite v2022](https://www.w3.org/TR/vc-di-eddsa/#multikey) specification for further details.
```

Multikey is a bit more general, so I suggest this:

```suggestion
    comment: Verification method to be used with, for example, data integrity proof cryptographic suites, such as the eddsa-2022 cryptographic suite. See the [EdDSA Cryptosuite v2022](https://www.w3.org/TR/vc-di-eddsa/#multikey) specification for further details.
```